### PR TITLE
.gitignore for OpenFrameworks.cc

### PR DESCRIPTION
**Pull request by C-F-4**
**Enhancement to the current gitignore repository**

Adds .gitignore for [OpenFrameworks](http://openframeworks.cc/)

Github Repository Link: [Github-OpenFrameworks](https://github.com/openframeworks/openFrameworks)

Here's a link to the official HomePage: **http://openframeworks.cc/**

### CODE REVIEW

**Comment by joeblau** [comment]
@C-F-4 - This tempalte is live on [gitignore.io](https://www.gitignore.io) - https://www.gitignore.io/api/openframeworks

**Comment by C-F-4** [comment]
@joeblau Is there a difference between the .gitignore files available on www.gitignore.io and https://github.com/github/gitignore?

**Comment by joeblau** [comment]
gitignore.io references github/gitignore though a submodule so there shouldn't be.  The main difference is that gitignore.io doesn't have strict requirements for what is considered a valid template therefore there are about 100 more templates.

This is still the master resource.

File: OpenFrameworks.gitignore [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+*.o
+[Dd]ebug*/
+[Rr]elease*/
+*.mode*
+*.app/
+*.pyc
+.svn/
+*.log
+
+# IDE files which should
```
Comment (shiftkey) [inline OpenFrameworks.gitignore]: We have existing templates for various IDEs that are up-to-date, and I don't want to take in changes which duplicate these rules.

Please isolate just the rules specific to OpenFrameworks, and add a comment if there are IDE templates that you think complement this template.

File: OpenFrameworks.gitignore [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+*.ilk
+*.aps
+ipch/
+
+# Eclipse
+.metadata
+local.properties
+.externalToolBuilders
+
+# operating system
```
Comment (shiftkey) [inline OpenFrameworks.gitignore]: We have existing templates for operating systems that are up-to-date, and I don't want to take in changes which duplicate these rules.

Please isolate just the rules specific to OpenFrameworks, and add a comment if there are OS templates under `Global` that you think complement this template.

**Pull request closed without merging**